### PR TITLE
Flush log messages

### PR DESCRIPTION
**Pull request by cajun-rat**
This might not be the best solution, but without it the systemd journal is useless.

Change-Id: I8c5f2ca0d76f69238bf21acbb7e224424ef8002e

### CODE REVIEW

**Comment by patrickvacek** [comment]
It's complaining about unformatted code.

File: src/logging.cc [hunk 1]
```diff
@@ -13,6 +13,7 @@ void logger_init() {
   gLoggingThreshold = boost::log::trivial::info;
   logging::add_console_log(std::cout, boost::log::keywords::format = "%Message%");
```
Comment (patriotyk) [inline src/logging.cc]: It is better to add true(auto_flush) as third parameter of `add_console_log`
http://www.boost.org/doc/libs/1_54_0/libs/log/doc/html/boost/log/add_console_lo_idp21543664.html

Comment (patriotyk) [reply src/logging.cc]: I think should be `logging::add_console_log(std::cout, boost::log::keywords::format = "%Message%", boost::log::keywords::auto_flush = true);`

Comment (patriotyk) [reply src/logging.cc]: Oh, it is already merged (((

Comment (cajun-rat) [reply src/logging.cc]: I still like your solution though

Comment (cajun-rat) [reply src/logging.cc]: I've opened a PR for your better solution: #598

**Pull request merged**